internal/service/ses/ses_cluster.go

```diff
@@ -344,15 +355,28 @@ func resourceNcloudSESClusterRead(ctx context.Context, d *schema.ResourceData, m
 	d.Set("login_key_name", cluster.LoginKeyName)
 	d.Set("manager_node_instance_no_list", cluster.ManagerNodeInstanceNoList)
 
-	searchEngineMap := d.Get("search_engine").([]interface{})[0].(map[string]interface{})
+	searchEngine := d.Get("search_engine").([]interface{})
 	searchEngineSet := schema.NewSet(schema.HashResource(ResourceNcloudSESCluster().Schema["search_engine"].Elem.(*schema.Resource)), []interface{}{})
-	searchEngineSet.Add(map[string]interface{}{
-		"version_code":   *cluster.SearchEngineVersionCode,
-		"user_name":      *cluster.SearchEngineUserName,
-		"user_password":  searchEngineMap["user_password"],
-		"port":           *cluster.SearchEnginePort,
-		"dashboard_port": *cluster.SearchEngineDashboardPort,
-	})
+
+	if len(searchEngine) == 0 { // API response not support user_password. Not currently available during import
+		searchEngineSet.Add(map[string]interface{}{
+			"version_code":   *cluster.SearchEngineVersionCode,
+			"user_name":      *cluster.SearchEngineUserName,
+			"user_password":  "",
+			"port":           *cluster.SearchEnginePort,
+			"dashboard_port": *cluster.SearchEngineDashboardPort,
+		})
+	} else { // Create exist in config
+		searchEngineMap := searchEngine[0].(map[string]interface{})
+		searchEngineSet.Add(map[string]interface{}{
+			"version_code":   *cluster.SearchEngineVersionCode,
+			"user_name":      *cluster.SearchEngineUserName,
+			"user_password":  searchEngineMap["user_password"],
+			"port":           *cluster.SearchEnginePort,
+			"dashboard_port": *cluster.SearchEngineDashboardPort,
+		})
+	}
```

Comment:
could we remove duplicated code, somehow like below)
```go
var userPassword string
if searchEngine, ok := d.GetOk("search_engine"); ok {
  searchEngineMap := searchEngine[0].(map[string]interface{})
  userPassword = searchEngineMap["user_password"]
}
searchEngineSet.Add(map[string]interface{}{
			"version_code":   *cluster.SearchEngineVersionCode,
			"user_name":      *cluster.SearchEngineUserName,
			"user_password":  userPassword,
			"port":           *cluster.SearchEnginePort,
			"dashboard_port": *cluster.SearchEngineDashboardPort,
		})
```

Reply:
fixed